docs/_developers/devguide.md

```diff
@@ -61,6 +61,19 @@ go install -v ./cmd/geth
 If you want to compile geth for an architecture that differs from your host, please
 consult our [cross compilation guide][cross-compile].
 
+### Building the Docker image
+
+Make sure you have [Docker Engine installed][docker-install]. To build the Docker image, run:
+```sh
+$ docker image build --build-arg "COMMIT=$(git rev-parse --short HEAD)" --tag ethereum/client-go:local .
```

Comment:
I only every do `docker build . -t my_tag`. Is `docker image build` and `docker build` identical? 
If so, `docker build` is shorter :)